src/pages/Admin/BackgroundSelect/BackgroundSelect.tsx

```diff
@@ -17,13 +17,15 @@ import "./BackgroundSelect.scss";
 
 export interface BackgroundSelectProps {
   venueName: string;
+  venueSlug: string;
```

Comment:
🌟 🪱 🌟 
`spaceSlug`?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2548/commits/3d30910bba970e8e3052fe6b53a34775a7b5ed05